trimmer_decorator.rb

```diff
@@ -0,0 +1,7 @@
+require_relative 'decorator'
+
+class TrimmerDecorator < Decorator
+  def correct_name
+    super[0..9]
+  end
```

Comment:
- [ ] To make this method work you shouldn't call super however you should use the nameable instance variable inherited from the decorator call correct_name and use the spread operator on that, something like:
```
    def correct_name
      @nameable.correct_name[0..9]
    end
```